### PR TITLE
docs: add build hang cause

### PR DESCRIPTION
**Pull request by aureamunoz**
@geoand  Could you please review this change?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -130,3 +130,15 @@ To fix the issue, you need to update the Maven importing settings:
 To import Quarkus you need to define the JVM Max Heap Size (E.g. `-Xmx1g`)
 
 **Note*:* As for now, we can't provide a unique Max Heap Size value. We have been reported to require from 768M to more than 3G to import Quarkus properly.
+
+* Build locks with DevMojoIT running infinitely
```
Comment (geoand) [inline CONTRIBUTING.md]: I suggest we say `Build hangs` instead of `Build locks`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -130,3 +130,15 @@ To fix the issue, you need to update the Maven importing settings:
 To import Quarkus you need to define the JVM Max Heap Size (E.g. `-Xmx1g`)
 
 **Note*:* As for now, we can't provide a unique Max Heap Size value. We have been reported to require from 768M to more than 3G to import Quarkus properly.
```
Comment (geoand) [inline CONTRIBUTING.md]: Should be `**Note**` ?

Comment (geoand) [reply CONTRIBUTING.md]: Oops, saw that this was not part of your change. Could you nevertheless fix this as well?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -130,3 +130,15 @@ To fix the issue, you need to update the Maven importing settings:
...
+* Build locks with DevMojoIT running infinitely 
+```
+./mvnw clean install
+# Wait...
+[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.192 s - in io.quarkus.maven.it.GenerateConfigIT
+[INFO] Running io.quarkus.maven.it.DevMojoIT
+
+```
+
+The DevMojoIT tests take a long time to run but too much time may not be normal. Make sure that nothing is running on 8080.
```
Comment (geoand) [inline CONTRIBUTING.md]: ` DevMojoIT tests take a long time to run but too much time may not be normal` -> ` DevMojoIT require a few minutes to run but anything more than that is not expected`

File: CONTRIBUTING.md (hunk 2 above)
Comment (aureamunoz) [reply CONTRIBUTING.md]: yes, of course

Comment (geoand) [reply CONTRIBUTING.md]: Thanks!

**Comment by geoand** [comment]
@aureamunoz can you please squash the commits and force push?

**Review by geoand — APPROVED** [review]
Thanks for this!

**Pull request merged**